wsmaster/che-core-api-factory-bitbucket-server/src/main/java/org/eclipse/che/api/factory/server/bitbucket/server/BitbucketPersonalAccessToken.java

```diff
@@ -11,23 +11,25 @@
  */
 package org.eclipse.che.api.factory.server.bitbucket.server;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import java.util.Objects;
 import java.util.Set;
 
-@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
could you clarify why the parsing rules have been changed?

Reply:
right, but is it relevant for this PR or this change is not related to the fix ?

Reply:
It is related, https://bitbucket-aws.crw-qe.com has been updated and some new fields are present in the API response.

Reply:
This bitbucket server instance is used by the QE team and that's why they reopened the issue.

Reply:
interesting, but will it work on the old version correctly?

Reply:
Why not? this annotation just ignores fields that are NOT defined in the DTO. If they are not defined so they are unnecessary for Che.